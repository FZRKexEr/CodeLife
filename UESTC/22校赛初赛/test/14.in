10 20
A 3 9 6
B 5
B 1
B 10
A 8 10 2
B 9
B 3
A 7 10 6
A 4 5 5
B 8
B 1
B 2
B 3
B 4
B 5
B 6
B 7
B 8
B 9
B 10
