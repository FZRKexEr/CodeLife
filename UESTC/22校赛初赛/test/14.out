3 9
1 2
10 10
8 9
3 7
8 9
1 2
1 2
3 3
4 5
4 5
6 6
7 7
8 9
8 9
10 10
